iib/workers/tasks/build.py

```diff
@@ -970,6 +970,17 @@ def handle_add_request(
                 catalog_from_index = get_catalog_dir(
                     from_index=from_index_resolved, base_dir=os.path.join(temp_dir, 'from_index')
                 )
+
+            # we have to remove all `deprecation_bundles` from `catalog_from_index`
+            # before merging catalogs otherwise if catalog was deprecated and
+            # removed from `index.db` it stays on FBC (from_index)
+            # Therefore we have to remove the directory before merging
+            for deprecate_bundle in deprecation_bundles:
+                # remove deprecated operators from FBC stored in index image
+                bundle_from_index = os.path.join(catalog_from_index, deprecate_bundle)
+                if os.path.exists(bundle_from_index):
+                    shutil.rmtree(bundle_from_index)
```

Comment:
This does not work - I am trying to find different solution.

Reply:
Might be solved by getting the name from LABEL 'operators.operatorframework.io.bundle.package.v1'